CHARTER.md

```diff
@@ -1,99 +1,31 @@
-# Smart Cart Project Charter
+Built by [Michael Fedell](https://github.com/michaelfedell), QA'd by [Finn Qiao](https://github.com/finnqiao)
 
-This project was started as a way to improve the grocery shopping experience, especially for young people who are still learning to cook. The project also serves as a testing ground for various tools and technologies such as cloud computing, machine-learning driven recommendation systems, application deployment and maintenance, as well as basic web development in the Flask-python ecosystem.
+# InstaCart
```

Comment:
Could add something at the end about how you would run all of this/execute the makefile